Telran_HomeWork/src/telran/Pizzeria20200413/AbstractDecorator.java

```diff
@@ -0,0 +1,20 @@
+package telran.Pizzeria20200413;
+
+//В данном примере Абстрактный класс не дает сильного выигрыша в объеме кода или функциональности
+public class AbstractDecorator implements PizzaInterface {
+    private PizzaInterface pizzaInterface;
+
```

Comment:
This abstract class contains a link to PizzaInterface. Without it you will need to refer the interface in every decorator.